testprojects/src/python/sources/BUILD

```diff
@@ -1,3 +1,3 @@
 python_library(
-  resources=globs('*.txt'),
+  sources=globs('*.py'),
```

Comment:
Good point. It doesn't require the resources dep any more. Regular resource deps are tested elsewhere in the relevant test, and the resources= wasn't actually tested at all (it was commented out with a comment explaining that the synthetic target wasn't available where it needed to be).